components/zoom_admin/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@pipedream/zoom_admin",
-  "version": "0.4.0",
+  "version": "0.4.1",
```

```suggestion
  "version": "0.5.0",
```
As per [our guidelines here](https://pipedream.com/docs/components/guidelines/#versioning), a new component definitely warrants a new `minor` version, rather than a `patch` version